PatternPal/PatternPal.Tests/New_TestCasesRecognizers/Bridge/BridgeTestCase05.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using Microsoft.AspNetCore.Components.Server.ProtectedBrowserStorage;
+using Console = System.Console;
+
+
+namespace PatternPal.Tests.New_TestCasesRecognizers.Bridge
+{
+
+    //This test is a possible bridge implementation.
+    /* Pattern:              Bridge
+     * Original code source:
+     *
+     * Requirements to fullfill the pattern:
+     *         Implementation interface or abstract class:
+     *            ✓  a) is an interface or abstract class <br/>
+     *            ✓  b) has at least one (abstract) method <br/>
```

Comment:
misschien deze requirement overal  (testcases + recognizer) veranderen in:
b) has at least one (if possible: abstract) method